CITATION.cff

```diff
@@ -4,57 +4,30 @@ authors:
   - family-names: Klar
     given-names:  Jochen
     orcid:        https://orcid.org/0000-0002-5883-4273
+
   - family-names: Michaelis
     given-names:  Olaf
     orcid:        https://orcid.org/0000-0002-5282-703X
-  - family-names: Engelhardt
-    given-names:  Claudia
-    orcid:        https://orcid.org/0000-0002-3391-7638
-  - family-names: Enke
-    given-names:  Harry
+
+  - family-names: Wallace
+    given-names:  David
+    orcid:        https://orcid.org/0000-0001-8958-4601
+
+  - family-names: Schröder
+    given-names:  Max
+    orcid:        https://orcid.org/0000-0003-1522-494X
+
+  - given-names:  Harry
```

Comment:
This name is now invalid. :)

Reply:
I don't get it.

Reply:
```
- given-names:  Harry
  given-names:  Enke
```

should be

```
- family-names:  Enke
  given-names:  Harry
```

Otherwise the cff is invalid.